model/command_response.go

```diff
@@ -18,17 +18,18 @@ const (
 )
 
 type CommandResponse struct {
-	ResponseType   string             `json:"response_type"`
-	Text           string             `json:"text"`
-	Username       string             `json:"username"`
-	ChannelId      string             `json:"channel_id"`
-	IconURL        string             `json:"icon_url"`
-	Type           string             `json:"type"`
-	Props          StringInterface    `json:"props"`
-	GotoLocation   string             `json:"goto_location"`
-	TriggerId      string             `json:"trigger_id"`
-	Attachments    []*SlackAttachment `json:"attachments"`
-	ExtraResponses []*CommandResponse `json:"extra_responses"`
+	ResponseType     string             `json:"response_type"`
+	Text             string             `json:"text"`
+	Username         string             `json:"username"`
+	ChannelId        string             `json:"channel_id"`
+	IconURL          string             `json:"icon_url"`
+	Type             string             `json:"type"`
+	Props            StringInterface    `json:"props"`
+	GotoLocation     string             `json:"goto_location"`
+	TriggerId        string             `json:"trigger_id"`
+	SkipSlackParsing string             `json:"skip_slack_parsing"`
```

Comment:
Why is this a `string`? The allowed values are `true` and `false` right? What do you think about making it a `bool` instead?

Reply:
done.